extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -595,7 +655,47 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
               visiblePercentageMin: 50,
               continuousTimeMin: 1,
             });
-    return superReturnValue;
+    if (this.isFluid_ && !this.element.style.width) {
+      setStyles(this.element, {
+        width: '100%',
+      });
+    }
+    return frameLoadPromise.then(result => {
+      if (this.isFluid_) {
+        // If this is a Fluid slot, we must send an initial message to
+        // safeframe.
+        this.connectFluidMessagingChannel();
+      }
+      return result;
+    });
+  }
+
+  /** @visibleForTesting */
+  connectFluidMessagingChannel() {
+    dev().assert(this.iframe.contentWindow,
+        'Frame contentWindow unavailable.');
+    this.iframe.contentWindow./*OK*/postMessage(
+        JSON.stringify(/** @type {!JsonObject} */
```

Comment:
nit: you can use `dict(` to create JsonObject

Reply:
Done.